DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/Index.cshtml

```diff
@@ -38,21 +42,19 @@
       </div>
     </div>
 
-    @if (Model.Courses.Any() && Model.CourseDetails != null) {
-      @if (!Model.CourseDetails.Active) {
+    @if (Model.Courses.Any() && Model.CourseDetails != null)
+    {
+      @if (!Model.CourseDetails.Active)
+      {

```

Comment:
This formatting change isn't in line with the rest of the project - it should stay as it is. Is there any way of setting these preferences in editor config / IDE?

Reply:
I know it's slightly strange because it's the opposite way round from C#.

Reply:
Not quite sure what happened here. Running the formatter puts this in line with the rest of the project, but I could've sworn I've run it elsewhere in the past where it's done the opposite. I'll keep an eye out in case the opposite occurs again, but I imagine I am just delusional.